mage/algorithms/traditional-graph-analytics/degree-centrality.md

```diff
@@ -0,0 +1,48 @@
+---
+id: degree-centrality-algorithm
+title: Degree Centrality
+sidebar_label: Degree Centrality
+---
+
+## Description
+
+**Degree Centrality** is a method for estimating the centrality of graph 
+entities.
+It is a simple centrality measure that counts how many neighbors a node has. 
+For a directed graph, we have two versions of the measure: in-degree and 
+out-degree.
+In-degree counts the number of in-coming edges, and out-degree counts the number
+of out-going edges.
+
+Usually, applications are more interested in in-degree measure, since in-coming
+edges are given by other nodes in a graph. Degree centrality can be used for any
+centrality use-case, such as social network or citation network analysis. More
+complex centrality measures are available when users need them.
```

Comment:
what do you mean by "More complex centrality measures are available when users need them? How and where are they available? How do they get them?